linux_os/guide/system/permissions/files/permissions_important_account_files/file_owner_etc_shadow.rule

```diff
@@ -0,0 +1,31 @@
+documentation_complete: true
```

Comment:
The same comment in the other file applies here, except renaming this one from `userowner_shadow_file.rule` -> `file_owner_etc_shadow.rule` and updating the corresponding fixes/ovals as above.

Reply:
Rules content are same in fact, I just copied userowner_shadow_file content as it works.

Reply:
@iokomin Yes, that's my point... the content is the same, so I'd suggest using the same file :) No point duplicating the same content in multiple places; that's why we're merging everything back into a single guide. Since most everything (the other rules in that folder for instance, but also the stuff in the `debian8` guide as well) uses the `file_` prefix, I'd prefer if the original source that you copied from (`userowner_shadow_file.rule`) was renamed to the name you chose here. Then we wouldn't have two, and you get to keep the more consistent name. :)